ship.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  shouldPrepare: ({ releaseType, commitNumbersPerType }) => {
+    const { fix = 0 } = commitNumbersPerType;
+    if (releaseType === 'patch' && fix === 0) {
+      return false;
+    }
+    return true;
+  },
+  publishCommand: ({ defaultCommand, tag }) =>
+    `${defaultCommand} --access public --tag next`,
```

Comment:
I couldn't find a way to let ship.js know that the tag should always be `next`, so I am just setting it here manually.